app/Http/Controllers/ArtistsController.php

```diff
@@ -40,7 +40,10 @@ public function show($id)
             ->where('visible', true)
             ->orderBy('id', 'desc')
             ->with(['tracks' => function ($query) {
-                $query->orderBy('display_order', 'ASC');
+                $query
+                    ->orderBy('display_order', 'asc')
+                    ->orderBy('exclusive', 'desc')
+                    ->orderBy('id', 'desc');
```

Comment:
this part hasn't been working after the `tracks.artist` below was added. Also this one probably should have id ascending instead.